admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/nav/template.vue

```diff
@@ -133,144 +142,170 @@
-      onHelpClick() {
-        $perAdminApp.action(this, 'showTour', '')
-      },
-      onTutorialsClick() {
-        document.getElementById('peregrine-main').classList.toggle('tutorial-visible')
-      },
-      onAboutClick() {
-        $('#aboutPeregrine').modal('open');
-      },
-      onSettingsClick() {
-        $perAdminApp.action(this, 'configureTenant', {
-          path: '/content/admin/pages/tenants/configure.html/content/'+this.state.tenant.name, name: this.state.tenant.name
-        })
-      },
-      onChangeWebsiteClick() {
-        $perAdminApp.action(this, 'selectPath', '/content/admin/pages/index')
-      },
-      onLogoutClick() {
-        window.location.href = '/system/sling/logout?resource=/index.html';
-      },
-      onLanguageClick() {
-        this.$refs.languageModal.open();
-      },
-      refreshTenants() {
-        this.tenants = $perAdminApp.getView().admin.tenants || []
-        this.state = $perAdminApp.getView().state
-      },
-      getActiveSection() {
-        const breadcrumbs = $perAdminApp.getView().adminPage.breadcrumbs
-        if (breadcrumbs) {
-          return breadcrumbs[0].path.split('/')[4]
+          target += `/path:/content/${this.state.tenant.name}`
         }
-        return 'welcome'
+      } else {
+        target = '/content/admin/pages/index'
+      }
+      return {
+        command: 'selectPath',
+        title: this.$i18n(section.title),
+        target
+      }
+    },
+    onSelectLang({name}) {
+      this.$i18nSetLanguage(name)
+      $perAdminApp.forceFullRedraw()
+    },
+    onSelectTenant(tenant) {
+      $perAdminApp.stateAction('setTenant', tenant)
+    },
+    onHelpClick() {
+      $perAdminApp.action(this, 'showTour', '')
+    },
+    onTutorialsClick() {
+      document.getElementById('peregrine-main').classList.toggle('tutorial-visible')
+    },
+    onAboutClick() {
+      $('#aboutPeregrine').modal('open');
+    },
+    onSettingsClick() {
+      $perAdminApp.action(this, 'configureTenant', {
+        path: '/content/admin/pages/tenants/configure.html/content/' + this.state.tenant.name,
+        name: this.state.tenant.name
+      })
+    },
+    onChangeWebsiteClick() {
+      $perAdminApp.action(this, 'selectPath', '/content/admin/pages/index')
+    },
+    onLogoutClick() {
+      window.location.href = '/system/sling/logout?resource=/index.html';
+    },
+    onLanguageClick() {
+      this.$refs.languageModal.open();
+    },
+    refreshTenants() {
+      this.tenants = $perAdminApp.getView().admin.tenants || []
+      this.state = $perAdminApp.getView().state
+    },
+    getActiveSection() {
+      const breadcrumbs = $perAdminApp.getView().adminPage.breadcrumbs
+      if (breadcrumbs) {
+        return breadcrumbs[0].path.split('/')[4]
+      }
+      return 'welcome'
+    },
+    copyUsername() {
+      this.$refs.usernameInput.setAttribute('type', 'text')
+      this.$refs.usernameInput.select()
+      try {
+        document.execCommand('copy')
```

Comment:
does this really work? I tried to copy the username and then paste it again later but had no success with that

Reply:
@reusr1 ill check again to make sure

Reply:
There was in-fact a problem with `display: none` I missed. fixed it though and also fixed checking if the copying did work